public/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link href='https://api.mapbox.com/mapbox-assembly/mbx/v0.18.0/assembly.min.css' rel='stylesheet'>
```

Comment:
We already added mapbox css under `src/assets/css` from `node_modules` if it the same css please remove it, if it is not please make sure it is not exist in `node_modules` before adding it from an external resource

Reply:
assembly.min.css seems to be a crucial part of the mapbox tooltip library. `node_modules` doesn't contain this file. Instead of taking it from CDN I've moved the content into lib folder under public folder with same name.